configure-oidc.md

```diff
@@ -75,3 +75,128 @@ When issuing a `cosign sign` command, add the flags to pass in the client secret
 ```
 cosign sign -y --fulcio-url=$FULCIO_URL --rekor-url=$REKOR_URL --oidc-issuer=$OIDC_ISSUER_URL $IMAGE --oidc-client-secret-file=<my secret filename> --oidc-client-id=<my client ID>
 ```
+
+## Github (federated via Keycloak)
+
+### Obtain Client ID and Secret from Github OAuth App
+
+To integrate RHTAS with Github as an identity provider, you must register your Github OAuth App in your Keycloak realm. 
+
+Locate your Github Oauth App, or create a new Github Oauth App following the directions [here](https://docs.github.com/en/apps/oauth-apps/building-oauth-apps/creating-an-oauth-app). When creating the OAuth App, it is fine to enter a placeholder value for the Authorization Callback URL field, as this field will need to be updated to hold a value from Keycloak after more setup steps are complete. The Homepage URL field can be any value.
+
+Locate the client ID and client secret values from the Oauth App page, as they will be required to create the custom resource file.
+
+### Log in to Keycloak console
+
+You will need to log in to the Keycloak console to complete some steps of this process. Once [RHTAS is installed](https://github.com/securesign/sigstore-ocp/blob/main/tas-easy-install.sh), you can do this by finding the route and admin password.
+
+```shell
+$ oc get routes -n keycloak-system | awk '/keycloak/ {print $2}' | head -1
+keycloak-keycloak-system.apps.rosa.umebj-fwkt3-8j8.y6tr.p3.openshiftapps.com
+```
+Copy this route, adding `https://` in front, into your browser.
```

Comment:
Do you mean the output of the command or the note about https? I've reworked it so please take a look (will push the new version shortly). I do think it's important to make a note about https in case they haven't done this before, as otherwise it will not work.

Reply:
Oh I see, your command adds that already. Neat! :tada: